packages/preferences-controller/src/PreferencesController.test.ts

```diff
@@ -113,113 +154,133 @@ describe('PreferencesController', () => {
-    const controller = new PreferencesController();
+    const controller = setupPreferencesController();
     controller.setSelectedAddress('0x95d2bc047b0ddec1e4a178eeb64d59f5e735cd0a');
     expect(controller.state.selectedAddress).toBe(
       '0x95D2bC047B0dDEc1E4A178EeB64d59F5E735cd0A',
     );
   });
 
   it('should set useTokenDetection', () => {
-    const controller = new PreferencesController();
+    const controller = setupPreferencesController();
     controller.setUseTokenDetection(true);
     expect(controller.state.useTokenDetection).toBe(true);
   });
 
   it('should set useNftDetection', () => {
-    const controller = new PreferencesController();
+    const controller = setupPreferencesController();
     controller.setOpenSeaEnabled(true);
     controller.setUseNftDetection(true);
     expect(controller.state.useNftDetection).toBe(true);
   });
 
   it('should set securityAlertsEnabled', () => {
-    const controller = new PreferencesController();
+    const controller = setupPreferencesController();
     controller.setSecurityAlertsEnabled(true);
     expect(controller.state.securityAlertsEnabled).toBe(true);
   });
 
   it('should set disabledRpcMethodPreferences', () => {
-    const controller = new PreferencesController();
+    const controller = setupPreferencesController();
     controller.setDisabledRpcMethodPreference('eth_sign', true);
     expect(controller.state.disabledRpcMethodPreferences.eth_sign).toBe(true);
   });
 
   it('should set isMultiAccountBalancesEnabled', () => {
-    const controller = new PreferencesController();
+    const controller = setupPreferencesController();
     controller.setIsMultiAccountBalancesEnabled(true);
     expect(controller.state.isMultiAccountBalancesEnabled).toBe(true);
   });
 
   it('should set showTestNetworks', () => {
-    const controller = new PreferencesController();
+    const controller = setupPreferencesController();
     controller.setShowTestNetworks(true);
     expect(controller.state.showTestNetworks).toBe(true);
   });
 
   it('should set isIpfsGatewayEnabled', () => {
-    const controller = new PreferencesController();
+    const controller = setupPreferencesController();
     controller.setIsIpfsGatewayEnabled(true);
     expect(controller.state.isIpfsGatewayEnabled).toBe(true);
   });
 
   it('should set showIncomingTransactions to false on ethereum network', () => {
-    const controller = new PreferencesController();
+    const controller = setupPreferencesController();
 
     controller.setEnableNetworkIncomingTransactions('0x1', false);
     expect(controller.state.showIncomingTransactions['0x1']).toBe(false);
   });
 });
+
+/**
+ * Setup a PreferencesController instance for testing.
+ *
+ * @param options - PreferencesController options.
+ * @returns A PreferencesController instance.
+ */
+function setupPreferencesController(
+  options: Partial<ConstructorParameters<typeof PreferencesController>[0]> = {},
```

Comment:
Nit: For some other controllers we have an `*Options` type that holds the options. Think it's worth it to extract that? I guess this is also fine too, just a bit verbose.

Reply:
Hmm, I actually find that pattern makes the code it bit harder to read. I prefer declaring options inline rather than in types declared somewhere else.